pkg/operator/sync.go

```diff
@@ -62,6 +72,97 @@ func (optr *Operator) syncAll(rconfig *renderConfig, syncFuncs []syncFunc) error
 	return agg
 }
 
+func (optr *Operator) syncRenderConfig(config *renderConfig) error {
```

Comment:
It's confusing that this function takes a `config` as a parameter but really it will be either the previous version or `nil`.  Maybe cleaner to separate it out of the other sync funcs array and have it take `()`?

Reply:
see your other comment